app/models/icon.rb

```diff
@@ -6,8 +6,8 @@ class Icon < ApplicationRecord
   belongs_to :image
   belongs_to :restore_to, :polymorphic => true
 
-  has_one :portfolio, :dependent => :destroy
-  has_one :portfolio_item, :dependent => :destroy
+  has_one :portfolio, :dependent => :nullify
```

Comment:
Will rubocop whine about this? Since we don't have a bot it isn't easy to see here.

Reply:
```
[ update_icon_relations ⚛ ] ~/git/manageiq/catalog-api $ rubocop app/models/icon.rb
Inspecting 1 file
.

1 file inspected, no offenses detected
```

Is this good enough :grin: